daemons/dss-checkout-sfn/app.py

```diff
@@ -12,23 +12,34 @@
 from chainedawslambda.s3copyclient import S3ParallelCopySupervisorTask
 from dss import chained_lambda_clients
 
-from dss.util.state_machine.checkout_states import state_machine_def
+from dss.stepfunctions.checkout.checkout_states import state_machine_def
 from dss.util.email import send_checkout_success_email, send_checkout_failure_email
 from dss.util.checkout import (parallel_copy, get_dst_bundle_prefix, get_manifest_files,
                                validate_file_dst, pre_exec_validate)
 
 app = domovoi.Domovoi()
-logger = dss.get_logger()
 
 dss.Config.set_config(dss.BucketConfig.NORMAL)
 email_sender = dss.Config.get_notification_email()
 default_checkout_bucket = dss.Config.get_s3_checkout_bucket()
-
 replica = "aws"
 
 for client_name, client_class in chained_lambda_clients():
     chainedawslambda.aws.add_client(client_name, client_class)
 
+logger = dss.get_logger()
```

Comment:
What's the motivation behind this code movement?

Reply:
@ttung Andrey requested the use of DSS logger instead of default logger

Reply:
Got it, make sense.

Reply:
This was already getting the dss logger...?  See L21.

Reply:
See last comment pls.

Reply:
Please don't move around code unless there's a reason to do so.